e2e/cypress/integration/account_settings/general/email_spec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @account_setting
```

Comment:
Should `setting` have an `s` on the end?  i.e. `settings`?

Reply:
None of the tests in there has the ending `s` so I would say no.

Reply:
Cool, thanks @streamer45 👍